packages/doenetml-worker-rust/lib-doenetml-core/src/core/general_state_var/boolean_state_var.rs

```diff
@@ -0,0 +1,226 @@
+use enum_dispatch::enum_dispatch;
+
+use crate::{components::prelude::*, dependency::DependencySource, ExtendSource};
+
+use super::util::{create_data_query_if_match_extend_source, string_to_boolean};
+
+/// A boolean state variable interface for calculating the value of a boolean variable from dependencies.
```

Comment:
Boolean state variable updaters? A little confusing since it is plural.

Reply:
Maybe rename `StateVarUpdaters` to `StateVarUpdater`?

Reply:
I used `Updaters` because there were multiple functions, but I am happy to have it singular.